scripts/pyspelling/custom_dictionary.txt

```diff
@@ -45,4 +52,5 @@ tradeoff
 transpilation
 Transpilation
 unlinkables
+winkler
```

Comment:
Should 'winkler' lowercase be included?

Reply:
I think you're correct... removing this entry.